### PR TITLE
wolfssl: update maintainers list in Portfile to have JacobBarthelmeh

### PR DESCRIPTION
**Pull request by JacobBarthelmeh**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7
Xcode 12.4

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by JacobBarthelmeh** [comment]
Follow up to this PR https://github.com/macports/macports-ports/pull/12818.

Only dependency could find with "git grep wolfssl | grep dep" was in the curl Portfile. Added in a revision number to the wolfssl variant but on my Mac it was not building. Possibly even without the revision changes. I may need your help for updating the one revision for the Portfile dependent on wolfSSL.

**Comment by reneeotten** [comment]
> Follow up to this PR #12818.
> 
> Only dependency could find with "git grep wolfssl | grep dep" was in the curl Portfile. Added in a revision number to the wolfssl variant but on my Mac it was not building. Possibly even without the revision changes. I may need your help for updating the one revision for the Portfile dependent on wolfSSL.

You need to add `revision 1` in the Portfile for `curl` likely just in the `wolfssl` variant

File: devel/wolfssl/Portfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ name                wolfssl
 version             5.0.0
 categories          devel security
 platforms           darwin
-maintainers         yassl.com:chris
+maintainers         {wolfssl.com:facts @JacobBarthelmeh}
```
Comment (reneeotten) [inline devel/wolfssl/Portfile:9]: can `openmaintainer` also be added so that minor changes to the Portfile can be made without your explicit permission?

**Comment by ryandesign** [comment]
> You need to add revision 1 in the Portfile for curl likely just in the wolfssl variant.

It is not possible to change the revision (or version or epoch) in a variant; `port outdated` does not accommodate that.

**Comment by ryandesign** [comment]
The first line of the commit message exceeds the recommended 50 character limit.

**Comment by reneeotten** [comment]
> It is not possible to change the revision (or version or epoch) in a variant; `port outdated` does not accommodate that.

thanks @ryandesign that's of course true... @JacobBarthelmeh can you please let us know if you're willing/able to make the requested changes so that we can finish this up?!

**Comment by reneeotten** [comment]
@JacobBarthelmeh we appreciate you willing to become the maintainer of this port! We do hope though that you're then also willing/able to acknowledge/respond to PRs, tickets, review requests. If you're not sure how to proceed here please let us know and I can make the few required changes this time. Thanks!

**Comment by JacobBarthelmeh** [comment]
Thanks @reneeotten , having the openmaintainer added for changes to the portfile seems good. I updated the commit message to make it a little shorter and added a commit for removing the duplicate line. Not sure know what needs to be updated in the cURL Portfile, could you make that change?

**Comment by ryandesign** [comment]
> Not sure know what needs to be updated in the cURL Portfile, could you make that change?

Nothing needs to be done in the curl port anymore because the curl port was already updated to a newer version the day after the wolfssl 5 update.

**Comment by ryandesign** [comment]
This commit removes `yassl.com:chris` from the maintainers line... has that person agreed to give up maintenance of this port? I didn't see a port abandonment ticket for this port (though I filed three tickets about this port over two years ago to which the maintainer never responded so it does seem to be abandoned).

**Comment by cconlon** [comment]
Hi @ryandesign, I was listed as the previous maintainer: `yassl.com:chris`.  I have approved transitioning the maintainer of this port over to @JacobBarthelmeh internally on our side.

**Pull request merged**